test/commons/color/get-background-color.js

```diff
@@ -743,13 +743,14 @@ describe('color.getBackgroundColor', function() {
 
 	it('should return the html canvas inherited from body bgColor when element content does not overlap with body', function() {
 		fixture.innerHTML =
-			'<div id="target" style="position: relative; top: 2px;">Text</div>';
+			'<div id="target" style="position: relative; top: 2px; height: 10px;">Text</div>';
```

Comment:
without the mocha styles on the page, this test would fail as the default div `bottom` rect was larger than the body `bottom` rect and would consider it overlapping. Just fixed to adjust the height of the div to be smaller than the body `bottom`.